<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miniphoto</class>
 <widget class="QWidget" name="miniphoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>miniphoto</string>
  </property>
  <widget class="QDockWidget" name="afterImgOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>11</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGraphicsView" name="Window2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>310</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSlider" name="window1S">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>151</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>201</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="basic_tab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>basic</string>
    </attribute>
    <widget class="QSlider" name="basicVal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>46</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>14</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleSDGothicNeoUL00</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="basicText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="resetbtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="effect_tab">
    <attribute name="title">
     <string>Effect</string>
    </attribute>
    <widget class="QCheckBox" name="invertCK">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>2</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>invert</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="grayscaleCK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>grayscale</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mosaicCK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>mosaic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>57</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>other</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>other</string>
     </property>
    </widget>
    <widget class="QSlider" name="mosaicVal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QRadioButton" name="blur_btn">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>171</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>blur</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sharpen_btn">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>171</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>sharpen</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Window1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="none_btn">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>171</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>11</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="resetAll">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>204</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
    </font>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>reset.png</normaloff>reset.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="view_btn">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>204</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
    </font>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>view.png</normaloff>view.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="p_btn">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>149</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>149</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoL00</family>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>321</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>open.png</normaloff>open.png</iconset>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>321</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>save.png</normaloff>save.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>308</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="window2S">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>20</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_btn">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>undo.png</normaloff>undo.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cropCK">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>32</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>crop.png</normaloff>crop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
